init/services/HestiaKERNEL/To_Unicode_From_String.sh

```diff
@@ -47,100 +46,22 @@ HestiaKERNEL_To_Unicode_From_String() {
         ___converted=""
         ___content="$1"
         while [ ! "$___content" = "" ]; do
-                # get byte value
-                ___byte="${___content%"${___content#?}"}"
-                ___content="${___content#${___byte}}"
+                ___codepoint="${___content%"${___content#?}"}"
+                ___content="${___content#${___codepoint}}"
+                ___codepoint="$(printf -- "%d" "'${___codepoint}")"
 
-                ## NOTICE
-                ## POSIX Shell splits control characters as 2 characters
-                ## instead of 1 (e.g. '\n' -> '\', 'n'). Hence, let's handle it
-                ## explictly. More info:
-                ##      https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap06.html
-                if [ "$___byte" = '\' ]; then
-                        # extract 1 more to get a full character
-                        ___byte="${___content%"${___content#?}"}"
-                        ___content="${___content#"$___byte"}"
-                        case "${___byte:-\\}" in
-                        "0")
-                                # '\0' null <NUL>
-                                ___byte=0
-                                ;;
-                        "a")
-                                # '\a' alert <BEL>
-                                ___byte=7
-                                ;;
-                        "b")
-                                # '\b' backspace <BS>
-                                ___byte=8
-                                ;;
-                        "t")
-                                # '\t' tab <HT>
-                                ___byte=9
-                                ;;
-                        "n")
-                                # '\n' newline <LF>
-                                ___byte=10
-                                ;;
-                        "v")
-                                # '\v' vertical-tab <VT>
-                                ___byte=11
-                                ;;
-                        "f")
-                                # '\f' form-feed <FF>
-                                ___byte=12
-                                ;;
-                        "r")
-                                # '\r' carriage-return <CR>
-                                ___byte=13
-                                ;;
-                        "e"|"E")
-                                # '\e' or '\E' escape <ESC>
-                                ___byte=27
-                                ;;
-                        *)
-                                # cancelled character - go for the next instead
-                                ___byte="$(printf -- "%d" "'${___byte}")"
-                                ;;
-                        esac
-                else
-                        ___byte="$(printf -- "%d" "'${___byte}")"
+                if [ $___codepoint -lt 0 ]; then
+                        ___codepoint=63 # encoder error - change to '?'
```

Comment:
It's confirmed that BSD and older shell system have problem converting string character into Unicode codepoint natively despite they're working in UTF-8 encoding format. For this kind of situation, any error will be replaced with "?".